### PR TITLE
Ollama: added method to delete a model

### PR DESCRIPTION
**Pull request by bidek**
## Change
Added a new method to `OllamaModels`. It is now possible to delete an Ollama model through the API.

## General checklist

- [X] There are no breaking changes
- [X] I have added unit and integration tests for my change
- [X] I have manually run all the unit and integration tests in the module I have added/changed, and they are all green
- [X] I have manually run all the unit and integration tests in the [core](https://github.com/langchain4j/langchain4j/tree/main/langchain4j-core) and [main](https://github.com/langchain4j/langchain4j/tree/main/langchain4j) modules, and they are all green

- [ ] I have added/updated the [documentation](https://github.com/langchain4j/langchain4j/tree/main/docs/docs)
- [ ] I have added an example in the [examples repo](https://github.com/langchain4j/langchain4j-examples) (only for "big" features)
- [ ] I have added/updated [Spring Boot starter(s)](https://github.com/langchain4j/langchain4j-spring) (if applicable)

## Checklist for adding new model integration

- [ ] I have added my new module in the [BOM](https://github.com/langchain4j/langchain4j/blob/main/langchain4j-bom/pom.xml)

## Checklist for adding new embedding store integration

- [ ] I have added a `{NameOfIntegration}EmbeddingStoreIT` that extends from either `EmbeddingStoreIT` or `EmbeddingStoreWithFilteringIT`
- [ ] I have added my new module in the [BOM](https://github.com/langchain4j/langchain4j/blob/main/langchain4j-bom/pom.xml)

## Checklist for changing existing embedding store integration

- [ ] I have manually verified that the `{NameOfIntegration}EmbeddingStore` works correctly with the data persisted using the latest released version of LangChain4j

### CODE REVIEW

File: langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaModels.java [hunk 1]
```diff
@@ -47,4 +47,16 @@ public Response<OllamaModelCard> modelCard(String modelName) {
         ), maxRetries);
         return Response.from(response);
     }
+
+    public void delete(OllamaModel ollamaModel) {
```
Comment (langchain4j) [inline langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaModels.java]:
```suggestion
    public void deleteModel(OllamaModel ollamaModel) {
```

File: langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaModels.java [hunk 2]
```diff
@@ -47,4 +47,16 @@ public Response<OllamaModelCard> modelCard(String modelName) {
...
+        delete(ollamaModel.getName());
+    }
+
+    public void delete(String ollamaModelName) {
```
Comment (langchain4j) [inline langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaModels.java]:
```suggestion
    public void deleteModel(String ollamaModelName) {
```

File: langchain4j-ollama/src/test/java/dev/langchain4j/model/ollama/OllamaClientIT.java [hunk 3]
```diff
@@ -48,4 +48,28 @@ void should_respond_with_model_information() {
         assertThat(modelDetailsResponse.getDetails().getFormat()).isEqualTo("gguf");
         assertThat(modelDetailsResponse.getDetails().getFamily()).isEqualTo("llama");
     }
+
+    @Test
+    void should_delete_model() {
```
Comment (langchain4j) [inline langchain4j-ollama/src/test/java/dev/langchain4j/model/ollama/OllamaClientIT.java:53]: Did you check if this test slows down all Ollama ITs?

**Review by ghost — COMMENTED** [review]
@bidek thank you!

File: langchain4j-ollama/src/test/java/dev/langchain4j/model/ollama/OllamaClientIT.java (hunk 3 above)
Comment (bidek) [reply langchain4j-ollama/src/test/java/dev/langchain4j/model/ollama/OllamaClientIT.java:53]: I ran all Ollama tests and did not see any slowdowns. All tests for Ollama run on my computer in about 1 minute. 50 sec is just for OllamaChatModelVisionIT

File: langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaModels.java (hunk 1 above)
Comment (bidek) [reply langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaModels.java]: fixed

File: langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaModels.java (hunk 2 above)
Comment (bidek) [reply langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaModels.java]: fixed

**Review by ghost — APPROVED** [review]
@bidek thanks!

**Pull request merged**